python3.6/PyZAPI/development.ini

```diff
@@ -0,0 +1,70 @@
+###
```

Comment:
We should not be committing our actual “development.ini” or “production.ini” files into the repository. We should craft a “developer.ini.example” if necessary. You don’t ever want to put in project specific or account specific items into GitHub (battlecry@) or the actual URL for your database in AWS. That exposes to the world where your database is sitting, and will provide an opportunity for someone to try and brute force themselves access.